codeReview/cr2/Practices/separation practice/main.cpp

```diff
@@ -0,0 +1,31 @@
+#include "EvenRandom.h"
+
+int main()
+{
+	EvenRandom r;
+	int nCount = 1; // ¦�� 10���� �����ϱ� ���� ī���� ����
```

Comment:
Неочевидні назви змінних, обьектів, зайві коментарі
```suggestion
	EvenRandom random;
	int numberCount = 1;
```